### PR TITLE
LPD-25542 Control panel visibility issue with user groups

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/LPD-25542

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

File: modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/UserUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+		List<Role> roles = UserUtil.getSiteRoles(_user);
+
+		Assert.assertEquals(roles.toString(), 0, roles.size());
+
+		_setSiteRole();
+
+		roles = UserUtil.getSiteRoles(_user);
+
+		Assert.assertEquals(roles.toString(), 1, roles.size());
+	}
```
Comment (andrea-ale-sbarra) [inline modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/UserUtilTest.java:141]: These tests look like are already verified with the first test, aren't they?

**Review by andrea-ale-sbarra — COMMENTED** [review]
A couple of questions

File: modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/UserUtilTest.java (hunk 1 above)
Comment (smottal) [reply modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/UserUtilTest.java:141]: The first test verify the all list but these are verifying the specific type (getSiteRoles, getOrganizationsRoles, getInheritedSiteRoles...)

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:report:2568980906

**Comment by smottal** [comment]
ci:forward:force

**Pull request closed without merging**